.gitignore

```diff
@@ -2,6 +2,7 @@
 .DS_Store
 .vscode
 morgan.log
+package-lock.json
```

Comment:
uhhhh we shouldn't be ignoring this file.
We should either use Yarn or Npm and stick to just one.
I vote Yarn :P

Reply:
Fixed